src/app/layout/qgslayoutpdfexportoptionsdialog.h

```diff
@@ -0,0 +1,77 @@
+/***************************************************************************
+                         qgslayoutpdfexportoptionsdialog.h
+                         -------------------------------------
+    begin                : August 2019
+    copyright            : (C) 2019 by Nyall Dawson
+    email                : nyall dot dawson at gmail dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSLAYOUTPDFEXPORTOPTIONSDIALOG_H
+#define QGSLAYOUTPDFEXPORTOPTIONSDIALOG_H
+
+#include <QDialog>
+#include "ui_qgspdfexportoptions.h"
+
+#include "qgsrendercontext.h"
+
+/**
+ * A dialog for customizing the properties of an exported PDF file.
```

Comment:
If one wants to be super picky, one would ask for a reference to layout export in this class description. I.e., '... properties of an exported layout PDF file' to give a bit more context to the reader (who would skip actually reading the QgsClassName :) )